apps/site/redirects.json

```diff
@@ -8,6 +8,14 @@
       "source": "/api.html",
       "destination": "/api"
     },
+    {
+      "source": "/api/corepack.html",
```

Comment:
The redirect, if implemented for latest, should be done after corepack is not available in that Node.js version
i.e. when v25.0.0 is released.

While corepack is included in the latest, it should highlight the decision in bold and provide a call to action, like it's suggested in https://github.com/nodejs/node/pull/57825